toolchest_client/tools.py

```diff
@@ -120,3 +122,31 @@ def kraken2(tool_args="", **kwargs):
         output_name="output.txt",
         **kwargs
     )
+
+
+def test(tool_args="", **kwargs):
+    """Run a test pipeline segment via Toolchest. A plain text file containing 'step one' is returned."
```

Comment:
Is the last quote a typo? Otherwise, everything else looks good.

Reply:
Whoops, yep. It's `success` now.